griptape/artifacts/image_artifact.py

```diff
@@ -19,6 +23,12 @@ class ImageArtifact(BlobArtifact):
     width: int = field(kw_only=True, metadata={"serializable": True})
     height: int = field(kw_only=True, metadata={"serializable": True})
 
+    def __attrs_post_init__(self) -> None:
+        # Generating the name string requires attributes set by child classes.
+        # This waits until all attributes are available before generating a name.
+        if self.name == self.id:
```

Comment:
As it currently stands, customers are (and were) expected to add the format to the name themselves, right?